docs/includes/keyboard-shortcuts-prerequisites.md

```diff
@@ -1,10 +1,8 @@
 > [!IMPORTANT]
-> Keyboard shortcuts are in preview. Please experiment with them in a development or testing environment but don't add them to a production add-in.
->
 > Keyboard shortcuts are currently only supported on Excel and only on these platforms and builds:
 >
->* Excel on Windows: Version 2010 (Build 13426.20000)
->* Excel on Mac: 16.43.1101.0
+>* Excel on Windows: Version 2012 (Build 13801.20000)
```

Comment:
Please double check this version. 13801 seems too late to be 2012. Isn't it 2102?

Reply:
You are correct, thanks for catching that! Fixed.